.devcontainer/Dockerfile

```diff
@@ -3,44 +3,45 @@
 
 # See here for image contents: https://github.com/microsoft/vscode-dev-containers/tree/v0.195.0/containers/cpp/.devcontainer/base.Dockerfile
 # [Choice] Debian / Ubuntu version (use Debian 11/9, Ubuntu 18.04/21.04 on local arm64/Apple Silicon): debian-11, debian-10, debian-9, ubuntu-21.04, ubuntu-20.04, ubuntu-18.04
```

Comment:
Could we also update the comment to mention Debian 12?

Reply:
Yes, I could update it too, thanks for the warning.